### DISCUSSION TITLE
clone_from still asking for a fingerprint confirmation.

### DISCUSSION THREAD

**Discussion by XNinety9** [gitpython-developers/GitPython#1592] [Q&A]
Hi All.

I use GitPython in a unit test's fixture: it clones a repo during setup to download a stub service I can test my client against.

When I execute the tests locally there is absolutely no issue: the repo is cloned as intended. I use an SSH key.

When I use a CI environment into which I don't have an SSH key, the during clone_from I get a confirmation message: "Are you sure you want to continue connecting (yes/no/[fingerprint])?".

Using when I've found so far in the project's issues, I came up with this code:

    with Git().custom_environment(
            GIT_SSH_COMMAND=
            'ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=accept-new -o http.sslVerify=false'
    ):
        _ = Repo.clone_from(
            url="ssh://git@bitbuckettsabdx.psc.tasfr.thales:7999/tsisbcube/b3_tools.git",
            to_path=temp_dir
        )

I'd like to skip the confirmation message, as the environment is, obviously, not interactive.

Thanks in advance for your help.

**Comment by Byron** [comment]
Overriding `GIT_SSH_COMMAND` seems like the way to control what `ssh` shows and what it doesn't show. How to do this exactly would a matter of which flags to provide to `ssh`. Please feel encouraged to share these here as well. Thank you.
